### PR TITLE
multimedia-tools: add alsamixer, bump mpg123 and squeezelite

### PR DESCRIPTION
**Pull request by awiouy**

### CODE REVIEW

File: packages/audio/alsa-utils/package.mk [hunk 1]
```diff
@@ -57,6 +56,9 @@ post_makeinstall_target() {
     rm -rf $INSTALL/usr/bin/$i
   done
 
+# keep alsamixer for the multimedia-tools addon
+  mv $INSTALL/usr/bin/alsamixer $ROOT/$PKG_BUILD/alsamixer/
```
Comment (lrusak) [inline packages/audio/alsa-utils/package.mk]: You shouldn't need to move it, just `rm $INSTALL/usr/bin/alsamixer`

Comment (awiouy) [inline packages/audio/alsa-utils/package.mk]: make install moves alsamixer, ie removes it from $BUILD/alsamixer
if we do this, alsamixer is lost

Comment (lrusak) [inline packages/audio/alsa-utils/package.mk]: Are you sure it doesn't copy it?

**Comment by awiouy** [comment]
@lrusak my bad: I was looking for alsamixer in $PKG_BUILD and alsamixer was in $PKG_BUILD/.$TARGET. Fixed.

**Pull request merged**